warehouse/integrations/mssql/mssql.go

```diff
@@ -151,7 +163,6 @@ func Connect(cred CredentialsT) (*sql.DB, error) {
 		Host:     net.JoinHostPort(cred.Host, strconv.Itoa(port)),
 		RawQuery: query.Encode(),
 	}
-	pkgLogger.Debugf("mssql connection string : %s", connUrl.String())
```

Comment:
This contains sensitive information like Passwords, we should not probably log it.